lib/res/default_settings.json

```diff
@@ -1,6 +1,5 @@
 {
     "spots": [],
-    "areas": [],
```

Comment:
shouldn't there be an empty "zones" array now?

Reply:
The Configuration.js code deals with undefined, so not necessary to have this entry.